modules/ding_event/ding_event.module

```diff
@@ -344,6 +357,24 @@ function ding_event_form_views_exposed_form_alter(&$form, &$form_state) {
     if (empty($input)) {
       $form['reset']['#access'] = FALSE;
     }
+
+    if (!empty($form['#info']['filter-term_node_tid_depth'])) {
+      $label = $form['#info']['filter-term_node_tid_depth']['label'];
+      unset($form['#info']['filter-term_node_tid_depth']['label']);
+      $form['event_category_term_node_tid_depth']['#title'] = '<h2 class="list-title sub-menu-title">' . $label . '</h2>';
+    }
+
+    if (!empty($form['#info']['filter-og_group_ref_target_id_entityreference_filter'])) {
+      $label = $form['#info']['filter-og_group_ref_target_id_entityreference_filter']['label'];
+      unset($form['#info']['filter-og_group_ref_target_id_entityreference_filter']['label']);
+      $form['og_group_ref_target_id_entityreference_filter']['#title'] = '<h2 class="list-title sub-menu-title">' . $label . '</h2>';
+    }
```

Comment:
https://inlead.atlassian.net/browse/DDBCMS-254

Reply:
I just tested this on normal distribution and I think it's already AJAX based? I don't see what this change does.

Reply:
Even-though the case is about ajax-fying things, this change seem to just wrap the filter heading with **h2** tags.

Reply:
Ok. Why is this needed?